openmdao/solvers/linear/direct.py

```diff
@@ -65,6 +65,45 @@ def format_singluar_error(err, system, mtx):
     return msg.format(system.pathname, loc_txt, varname)
 
 
+def format_singluar_csc_error(system, matrix):
+    """
+    Format a coherent error message when the CSC matrix is singular.
+
+    Parameters
+    ----------
+    system : <System>
+        OpenMDAO system containing the Directsolver.
```

Comment:
is it necessary to qualify this as an `OpenMDAO` system?  seems to be clear by context.. 
(don't want to have to add `OpenMDAO` to all our docstrings...)

Reply:
fixed